skills/csharp/tests/phoneskill.tests/Flow/InterruptionTests.cs

```diff
@@ -39,5 +40,40 @@ await GetTestFlow()
                .AssertReply(Message(PhoneMainResponses.CancelMessage))
                .StartTestAsync();
         }
+
+        [TestMethod]
+        public async Task Test_Action_Help_Interruption()
+        {
+            var activity = new Activity()
+            {
+                Type = ActivityTypes.Event,
+                Name = "OutgoingCall",
+            };
+
+            await GetSkillTestFlow()
+               .Send(activity)
+               .AssertReply(Message(OutgoingCallResponses.RecipientPrompt))
+               .Send(GeneralUtterances.Help)
+               .AssertReply(Message(PhoneMainResponses.HelpMessage))
+               .AssertReply(Message(OutgoingCallResponses.RecipientPrompt))
+               .StartTestAsync();
+        }
+
+        [TestMethod]
+        public async Task Test_Action_Cancel_Interruption()
+        {
+            var activity = new Activity()
+            {
+                Type = ActivityTypes.Event,
+                Name = "OutgoingCall",
+            };
+
+            await GetSkillTestFlow()
+               .Send(activity)
+               .AssertReply(Message(OutgoingCallResponses.RecipientPrompt))
+               .Send(GeneralUtterances.Cancel)
+               .AssertReply(Message(PhoneMainResponses.CancelMessage))
+               .StartTestAsync();
```

Comment:
check ActionResult here? Same for Test_Action_Help_Interruption

Reply:
fixed.